src/main/java/org/ow2/proactive/addons/email/EmailSender.java

```diff
@@ -125,57 +107,45 @@
 
     protected String from;
 
-    protected String host;
-
-    protected String username;
-
-    protected String password;
-
     protected String subject;
 
-    protected String trustSsl = "*";
-
     protected String fileToAttach;
 
     protected String fileName = "attachment.txt";
 
-    protected EmailSender(boolean auth, boolean debug, boolean enableStartTls, int port, List<String> cc,
-            List<String> bcc, List<String> recipients, String body, String from, String host, String username,
-            String password, String subject, String trustSsl, String fileToAttach, String fileName) {
-        this.auth = auth;
+    protected Properties properties;
+
+    protected EmailSender(Properties properties, List<String> cc, List<String> bcc, List<String> recipients,
+            String body, String from, String subject, String fileToAttach, String fileName) {
+        this.properties = properties;
         this.bcc = bcc;
         this.body = body;
         this.cc = cc;
-        this.debug = debug;
-        this.enableStartTls = enableStartTls;
         this.from = from;
-        this.host = host;
-        this.password = password;
-        this.port = port;
         this.recipients = recipients;
         this.subject = subject;
-        this.trustSsl = trustSsl;
-        this.username = username;
         this.fileToAttach = fileToAttach;
         this.fileName = fileName;
 
         checkInstanceFieldsConsistency();
     }
 
+    private void setProperties(Properties properties) {
```

Comment:
I believe this method is not used any more

Reply:
Yes you are right, I'll remove it